cloud_AWS/terraform/module/examples/multiple-vpc-regions/input_data.json

```diff
@@ -0,0 +1,18 @@
+{
+  "external_id" : "",
+  "plan_id" : "",
```

Comment:
better: 
"plan_id" : "<plan_id>", so it doesnt suggest an empty string can be used here

Reply:
Implemented suggestion

Reply:
Changed to INI file but still to do

Reply:
Resolved